### PR TITLE
Adding PulsarAdmin configs to PulsarHelper

### PR DESCRIPTION
**Pull request by ericm-db**
### Motivation
In #151 we added `admin.url` without adding Admin Confs. 

### Modifications

*Describe the modifications you've done.*

Basically reverting #120 
### Verifying this change

- [ x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

- [ x] This change is a trivial rework / code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:

- [ ] This change added tests and can be verified as follows:

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required`
- [x] `no-need-doc`
- [ ] `doc`

### CODE REVIEW

**Comment by ericm-db** [comment]
@chaoqin-li1123 For some reason I can't add you as a reviewer

File: src/main/scala/org/apache/spark/sql/pulsar/PulsarProvider.scala [hunk 1]
```diff
@@ -258,6 +261,20 @@ private[pulsar] object PulsarProvider extends Logging {
     }
   }
 
+  private def getAdminParams(parameters: Map[String, String]): Map[String, String] = {
```
Comment (chaoqin-li1123) [inline src/main/scala/org/apache/spark/sql/pulsar/PulsarProvider.scala:264]: Why don't we use getModuleParams() here?

**Comment by chaoqin-li1123** [comment]
Code change LGTM, Thanks for the work! 
Could you fix the scala style error?
Also please update the doc that admin config need to be provided for admission control to work.

**Comment by chaoqin-li1123** [comment]
Can we add a small test for this change? Set one of the  pulsar.admin.* option in the query, if this option is invalid, an exception should be thrown.

**Review by chaoqin-li1123 — APPROVED** [review]
LGTM, thanks!

**Review by nlu90 — APPROVED** [review]

**Pull request merged**